### PR TITLE
feat:list component added

### PR DESCRIPTION
**Pull request by KanekarSnehal**
Following types of lists are added
1)ordered list
2)unordered list
3)list with bullets
4)List with inline items
list

### CODE REVIEW

File: components/list.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+.list-style-circle {
+    list-style-type: circle;
+}
+
+.list-style-square {
+    list-style-type: square;
+}
+
+.list-style-circle {
```
Comment (SukanyaSen530) [inline components/list.css:16]: Repeated the same class name

Comment (KanekarSnehal) [reply components/list.css]: done ```ecefa320c3d57531123df75a6abdbfee5a228010```

**Review by SukanyaSen530 — APPROVED** [review]
Looks good to me :)

**Review by nerdyspook — APPROVED** [review]
It looks good, you can merge.

**Review by tewarimohit — COMMENTED** [review]
Nicely Documented. LGTM.

**Pull request merged**